.github/workflows/windows_build.yml

```diff
@@ -0,0 +1,68 @@
+name: Windows Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
We will need to see if we want to keep this for every PR, or only on merge. But out of scope, lets look into this in a new PR.